File: Concurs Inot Java/ProbaDTO.java
package me.model;

import java.io.Serializable;

public class ProbaDTO implements Serializable {
    private Integer id;
    private float distanta;
    private Stiluri stil;
    private int nrParticipanti;

    public ProbaDTO(Integer id, float distanta, Stiluri stil, int nrParticipanti) {
        this.id=id;
        this.distanta = distanta;
        this.stil = stil;
        this.nrParticipanti=nrParticipanti;
    }

    public Integer getId() {
        return id;
    }

    public float getDistanta() {
        return distanta;
    }

    public Stiluri getStil() {
        return stil;
    }

    public int getNrParticipanti() {
        return nrParticipanti;
    }
}

File: Concurs Inot C#/ProbaDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problema_3_CSharp.me.model
{
    [Serializable]
    public class ProbaDTO
    {
        public int Id { get; }
        public float Distanta { get; }
        public Stiluri Stil { get; }
        public int NrParticipanti { get; }

        public ProbaDTO(int id, float distanta, Stiluri stil, int nrParticipanti)
        {
            Id = id;
            Distanta = distanta;
            Stil = stil;
            NrParticipanti = nrParticipanti;
        }
    }
}
